### PR TITLE
Update favicons and theme

### LINKED ISSUE #68: Update Favicons
Update Favicons to use the new ones from this PR: https://github.com/LibertyDSNP/frequency-xyz/pull/11

### PR DESCRIPTION
**Pull request by wilwade**
- Needed new Favicons Closes #68 
- Theme was out of date. Updated, and reapplied customization
- No content changes.

## How to Test

- Pull
- Follow [install](https://github.com/LibertyDSNP/frequency-docs/blob/main/README.md#install) instructions
- `npm run serve` (Favicon doesn't work with watch)
- See Favicon (feel free to check the `head` for the other favicons)
- See that the theme looks the same
    - Some tiny updates like the new sidebar indicator

## Screenshots

image

image
image

### CODE REVIEW

File: theme/index.hbs [hunk 1]
```diff
@@ -20,12 +20,12 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta name="theme-color" content="#ffffff" />
 
-        <link rel="icon" href="{{ path_to_root }}/favicon.ico" sizes="any" />
```
Comment (wilwade) [inline theme/index.hbs:23]: Moved to `./head.hbs`

File: theme/tomorrow-night.css
Comment (wilwade) [inline theme/tomorrow-night.css:1]: Theme update

File: theme/css/print.css
Comment (wilwade) [inline theme/css/print.css:1]: Theme update

File: theme/css/general.css
Comment (wilwade) [inline theme/css/general.css:1]: Basically all theme update

File: theme/css/chrome.css
Comment (wilwade) [inline theme/css/chrome.css:1]: Basically all theme update

File: theme/clipboard.min.js
Comment (wilwade) [inline theme/clipboard.min.js:1]: Pulling in to keep the version aligned with the rest of the theme.

File: theme/book.js
Comment (wilwade) [inline theme/book.js:1]: Basically all theme update

**Review by JoeCap08055 — APPROVED** [review]
* Checkout out branch
* Verified favicon & theme

**Pull request merged**